stage0/src/main.rs

```diff
@@ -63,12 +67,12 @@ static SEV_SECRETS: MaybeUninit<oak_sev_guest::secrets::SecretsPage> = MaybeUnin
 #[no_mangle]
 static SEV_CPUID: MaybeUninit<oak_sev_guest::cpuid::CpuidPage> = MaybeUninit::uninit();
 
-/// The default entry point for the kernel if one wasn't supplied via the QEMU fw_cfg device.
-const DEFAULT_KERNEL_ENTRY: u64 = 0x200000;
-
 /// We create an identity map for the first 1GiB of memory.
 const TOP_OF_VIRTUAL_MEMORY: u64 = Size1GiB::SIZE;
 
+/// The file path used by Stage0 to read the kernel command-line from the fw_cfg device.
+const CMDLINE_FILE_PATH: &[u8] = b"opt/stage0/cmdline\0";
```

Comment:
I think the cmdline stuff should be in `kernel.rs` as well.

Reply:
Moved the code.